server/controllers/devices.coffee

```diff
@@ -93,7 +93,9 @@ initAuth = (req, cb) ->
     [username, password] = extractCredentials req.headers['authorization']
     # Initialize user
     user = {}
-    user.body = password: password
+    user.body =
+        username: username
+        password: password
```

Comment:
`initAuth` is used by several functions. Are you sure your change has no side effect on them?